File: Assets/Scripts/Generate.js
#pragma strict

public var objects : GameObject[];

function Start () {
	InvokeRepeating("CreateObstacle", Score.obstacleSpeedOffset, Score.obstacleSpeed);
}

function CreateObstacle () {
	var item = objects[Mathf.Floor(Random.value * objects.length)];
	Instantiate(item);
}

File: Assets/Scripts/Generate.cs

/*
using UnityEngine;
using System.Collections;

public class Generate : MonoBehaviour {
	
	public GameObject[] objects;
	
	void Start () {
		InvokeRepeating("CreateObstacle", Score.obstacleSpeedOffset, Score.obstacleSpeed);
	}

	void CreateObstacle () {
		float i = Random.value * objects.Length;
		GameObject item = objects[(int)Mathf.Floor(i)];
		Instantiate(item);
	}
}
*/